apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/RdaServerJobIT.java

```diff
@@ -112,30 +115,30 @@ public void testS3() throws Exception {
             new FissClaimStreamCaller(new FissClaimTransformer(clock, hasher));
         final GrpcResponseStream<RdaChange<PreAdjFissClaim>> fissStream =
             fissCaller.callService(fissChannel, CallOptions.DEFAULT, 1098);
-        assertEquals(true, fissStream.hasNext());
+        assertTrue(fissStream.hasNext());
```

Comment:
FYI assertEquals here was a deliberate choice since it provides a more meaningful message when the test fails than just "assertion failed".  But it's not worth rolling all of them back not that you've changed them

Reply:
I just tested the error message with junit 5 and it's the more useful message now (expected true but was false) so these changes are fine.  They'll eliminate some complaints from IDEA as well.